projects/vdk-plugins/quickstart-vdk/.plugin-ci.yml

```diff
@@ -31,6 +31,32 @@ build-py310-quickstart-vdk:
   image: "python:3.10"
 
 
+quickstart-vdk-release-acceptance-test:
+  variables:
+    PLUGIN_NAME: quickstart-vdk
+  stage: pre_release
+  before_script:
+    - cd projects/vdk-plugins/quickstart-vdk
+  script:
+    - set -x
+    - pip install -U pip
+    - python setup.py sdist --formats=gztar
+    - pip install dist/* --extra-index-url $PIP_EXTRA_INDEX_URL
+    - pip install vdk-heartbeat --extra-index-url $PIP_EXTRA_INDEX_URL
+    - pip list
+    - export VDKCLI_OAUTH2_REFRESH_TOKEN=$VDK_API_TOKEN
+    - export VDK_HEARTBEAT_OP_ID="quickstart-vdk-$CI_PIPELINE_ID"
+    - export JOB_NAME=quickstart-vdk-release-test-job-$(date +%s)
+    - vdk-heartbeat -f cicd/release_acceptance_test_heartbeat_config.ini
```

Comment:
Should the heartbeat use the local k8s setup bootstrapped by quickstart-vdk for the purposes of testing it?

Reply:
I have configured it to run the test against the demo control service installation. I think quickstart-vdk is configured to run against a local control service by default. Do you mean to run a control service locally and then run the test against it?

Reply:
The only `release_acceptance_test_heartbeat_config.in` file found in the project is https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-core/cicd/release_acceptance_test_heartbeat_config.ini#L5 

It is btw not in cicd/ dir that is relative to `cd projects/vdk-plugins/quickstart-vdk`.

Also, it points to the demo installation the way you explained. At the same time, the purpose of quickstart-vdk is to bootstrap a local environment instead of a remote one like the demo one. 
Therefore, the purpose of heartbeat tests in the context of vdk-quickstart, will be running in that newly bootstrapped environment, to verify operability.

Reply:
Yes, it seems that I was mistakenly thinking that the purpose of this test is to check if vdk-core specifically is functional when releasing quickstart-vdk. As I see it, I should run "vdk server --install" in the CI/CD job before running vdk-heartbeat and remove all configuration values pointing to the demo installation. Does that sound right?

Reply:
> Yes, it seems that I was mistakenly thinking that the purpose of this test is to check if vdk-core specifically is functional when releasing quickstart-vdk
Seems about right to me, the purpose is to check vdk-core in the environment bootstrapped by quickstart-vdk. The operability in demo environment is checked on vdk-core release itself, correct?

Reply:
Yes, vdk-core tests run against the demo installation. Thank you!

Reply:
`vdk server --install` would not work in CICD since it requires [kind](https://kind.sigs.k8s.io/) which cannot be easily installed. So for lack of better option, using the demo environment seems fine to me.